### PR TITLE
OpenML workshop blog post

### PR DESCRIPTION
**Pull request by HeidiSeibold**

### CODE REVIEW

File: _source/2016-09-09-mlr-loves-OpenML.Rmd [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  makeLearner("classif.ksvm", type = "spoc-svc")
+)
+```
+
+- Run the learners on the three tasks.
+```{r}
+grid = expand.grid(task.id = taskinfo$task.id, 
+                   lrn.ind = seq_along(lrn.list))
+
+runs = lapply(seq_row(grid), function(i) {
```
Comment (jakobbossek) [inline _source/2016-09-09-mlr-loves-OpenML.Rmd:97]: `seq_row` is a BBmisc function!

**Comment by HeidiSeibold** [comment]
Is that a problem?

File: _source/2016-09-09-mlr-loves-OpenML.Rmd (hunk 1 above)
Comment (HeidiSeibold) [reply _source/2016-09-09-mlr-loves-OpenML.Rmd:97]: Is that a problem?

Comment (berndbischl) [reply _source/2016-09-09-mlr-loves-OpenML.Rmd:97]: not sure what you guys are talking about here? can we make the sentences a bit longer and precise? maybe with active hints?

**Comment by jakob-r** [comment]
The images should go into a directory for this blog post `./images/2016-09-09-mlr-loves-OpenML/`

File: _source/2016-09-09-mlr-loves-OpenML.Rmd (hunk 1 above)
Comment (jakob-r) [reply _source/2016-09-09-mlr-loves-OpenML.Rmd:97]: `library(BBmisc)` is missing.

Comment (HeidiSeibold) [reply _source/2016-09-09-mlr-loves-OpenML.Rmd:97]: Is there now

**Comment by jakob-r** [comment]
Perfect, now you also have to [merge the upstream master](https://help.github.com/articles/syncing-a-fork/). And then we are ready to go?
Okay - we could also do this manually ourselves to merge the conflict.

**Comment by HeidiSeibold** [comment]
Thanks for the help :)

Do I need to render as well? If so, can I use the make file in _source and just go `make all`?

**Comment by jakob-r** [comment]
Yes you have to render the Rmd to a md file. Just call `make` in _source and the file will be generated.

**Comment by HeidiSeibold** [comment]
Uff, I do not have all gems installed. Can someone render for me? I am really busy this week, but this post should be published soon...

**Comment by PhilippPro** [comment]
I rendered it and published the post. I also added you to the _config.yml file, so you are correctly written as author.

**Comment by HeidiSeibold** [comment]
@berndbischl @giuseppec @jakobbossek @ja-thomas, please feel free to still change stuff.

**Pull request merged**